charts/stable/mealie/templates/_secrets.tpl

```diff
@@ -89,7 +89,7 @@ api:
     {{- with $api.ldap.id_attribute }}
     LDAP_ID_ATTRIBUTE: {{ . | quote }}
     {{- end -}}
-    {{- with $api.ldap.name_attribute }}
```

Comment:
Lets instead update values.yaml and questions.yaml to use the `name_attribute`.
As this will match the variable name and avoid any confusions